src/commands/project/create-variable-group.decorator.json

```diff
@@ -9,7 +9,7 @@
       "required": true
     },
     {
-      "arg": "-d, --hld-repo-url <hld-repo-url>",
+      "arg": "-U, --hld-repo-url <hld-repo-url>",
```

Comment:
why is this uppercase U?

Reply:
cannot use -d because -d is for --devops-project. So I have to choose a new one. The plan to use -U for all url related options. And cannot use -u because -u is for user name or user id, sp id related command.